terraform/modules/jaspersoft/box.tf

```diff
@@ -0,0 +1,62 @@
+resource "aws_security_group" "jaspersoft_server" {
+  vpc_id      = var.vpc_id
+  description = "Jaspersoft server instance"
+}
+
+# tfsec:ignore:aws-vpc-no-public-egress-sgr
+resource "aws_security_group_rule" "jaspersoft_server_egress" {
+  type              = "egress"
+  description       = "Allow all egress"
+  from_port         = 0
+  to_port           = 0
+  protocol          = -1
+  cidr_blocks       = ["0.0.0.0/0"]
+  security_group_id = aws_security_group.jaspersoft_server.id
+}
+
+resource "aws_security_group_rule" "jaspersoft_server_ssh_ingress" {
+  type                     = "ingress"
+  from_port                = 22
+  to_port                  = 22
+  protocol                 = "tcp"
+  source_security_group_id = var.allow_ssh_from_sg_id
+  description              = "SSH from bastion"
+  security_group_id        = aws_security_group.jaspersoft_server.id
+}
+
+resource "aws_security_group_rule" "jaspersoft_server_http_ingress" {
+  type                     = "ingress"
+  from_port                = 8080
+  to_port                  = 8080
+  protocol                 = "tcp"
+  source_security_group_id = aws_security_group.alb.id
+  description              = "HTTP on 8080 from ALB"
+  security_group_id        = aws_security_group.jaspersoft_server.id
+}
+
+resource "aws_instance" "jaspersoft_server" {
+  ami           = "ami-00f499a80f4608e1b" #	eu-west-1 bionic 18.04 LTS amd64
```

Comment:
Can you use the [aws_ami](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ami) data source to look up the latest version? So we automatically get maintenance updates

Reply:
I'm hesitant to do this since this will have a database on that we don't want to recreate it all the time.

It does have unattended-upgrade set up, though that will not include updates requiring a restart by default.

Would you like me to switch to it anyway and see where we end up with the database?

Reply:
Hm, the database is a fair point. Sure, leave it as is.

Not a huge deal as jaspersoft is something that we're considering moving away from next year.